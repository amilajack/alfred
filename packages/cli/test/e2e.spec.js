import Joker from '@amilajack/joker';

describe('e2e', () => {
  it('should get help commands', async () => {
    await new Joker()
      .base(require.resolve('../lib/alfred.js'))
      .run('help')
      .expect(result => {
        expect(result.stdout).toMatchSnapshot();
        expect(result.stderr).toEqual('');
      })
      .code(0)
      .end();
  });

  it.skip('should create new project', () => {});

  it.skip('should learn new alfred skill', () => {});
});
