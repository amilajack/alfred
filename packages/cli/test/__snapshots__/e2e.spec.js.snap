// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`e2e should get help commands 1`] = `
"Usage: alfred [options] [command]

A Modular JS Toolchain

Options:
  -v, --version       output the version number
  -h, --help          output usage information

Commands:
  new <project-name>  Create a new Alfred project
  learn <skill>       Add an Alfred skill to your project
  skills              List all the subcommands and skills
  run <skill>         Run an Alfred skill
  migrate [glob]      Migrate to the latest version of ES
  help [cmd]          display help for [cmd]"
`;
